networkx/algorithms/centrality/current_flow_betweenness.py

```diff
@@ -29,14 +29,13 @@ def approximate_current_flow_betweenness_centrality(
     weight=None,
     dtype=float,
     solver="full",
-    epsilon=0.5,
-    kmax=10000,
+    k=None,
     seed=None,
```

Comment:
Let's make the new keyword arg keyword only.  And we should have `seed` keyword only also -- because it changes position in the function signature when we replace `epsilon` and `kmax` with `k`.  So, anyone using position args will need to change their code -- and they should make the call use keywords.

So, I think something like this should work:
```
*, 
k=None,
seed=None,
```
Also -- changing the argument structure will require that the random_state decorator get updated too. 
```python
@py_random_state("seed")
```